perceiver_ar/experiment.py

```diff
@@ -42,6 +42,8 @@
 from perceiver_ar import perceiver_ar_model
 import tensorflow as tf
 
+tf.config.set_visible_devices([], "GPU")
```

Comment:
Can you add a comment above this line. Something like:

```
# Ensure TensorFlow doesn't use GPU.
```

Reply:
I agree and added the comment.